vkcs/db/resource_instance.go

```diff
@@ -513,8 +513,9 @@ func resourceDatabaseInstanceCreate(ctx context.Context, d *schema.ResourceData,
 	}
 
 	if replicaOf, ok := d.GetOk("replica_of"); ok {
-		if createOpts.Datastore.Type == PostgresPro {
-			return diag.Errorf("replica_of field is forbidden for PostgresPro")
+		err := checkReplicaDatastore(createOpts.Datastore.Type)
```

Comment:
I think supported datastores should also be displayed in the doc for "replica_of" attribute.

Reply:
Added doc description generation based on these slices